thor/bytes32.go

```diff
@@ -23,6 +23,11 @@ var (
 	_ json.Unmarshaler = (*Bytes32)(nil)
 )
 
+var (
+	ErrInvalidPrefix = errors.New("invalid prefix")
```

Comment:
What's the purpose of exporting them?

Reply:
Because I want to check the error equality based on the error object instead of comparing strings, like [here](https://github.com/paologalligit/thor/blob/test_coverage/api/transactions/transactions_test.go#L132)

Reply:
If it's for test purposes only, I would suggest string-based comparision is fine.

Reply:
ok, done 👍